Doc/library/json.rst

```diff
@@ -267,8 +267,9 @@ Basic Usage
    kwarg; otherwise :class:`JSONDecoder` is used.  Additional keyword arguments
    will be passed to the constructor of the class.
 
-   If the data being deserialized is not a valid JSON document, a
-   :exc:`JSONDecodeError` will be raised.
+   If the data being deserialized is not correctly encoded, a
```

Comment:
It should also be added for `loads()`.

`UnicodeDecodeError` can only be raised for binary files. `loads()` for text never raise `UnicodeDecodeError`, and in case of `load()` for text file, `UnicodeDecodeError` and `OSError` can only be raised by the `read()` of the text file. There are many details, and without explaining them we will always get complqains about incomplete documentation, but adding all details will make the documentation too overloaded. We usually do not specify all exceptions which can be raised. Is there a good comromize?